### DISCUSSION TITLE
movie_extpipe from netcam_high_url?

### DISCUSSION THREAD

**Discussion by feelingwalnut** [Motion-Project/motion#1826] [Q&A] [answered]
i have been unable to find a way to use movie_extpipe with the full resolution the high_url gives us. i've tried a few variants even using ffmpeg to call an rtsp feed but all have either been non-functional or problematic.

any one know what the "official" way is? is it just a matter of selecting a different pipe? how does it work on the normal movie capture option?

**Comment by tosiara** [comment]
What exactly does not work for you? What error do you get? Do you have full log?

**Comment by feelingwalnut** [comment]
no errors, just unsure of how to configure the parameter.

with these settings you will get a video from the netcam_url @480x360
netcam_high_url rtsp://user:pass@ip:port/Streaming/channels/101
netcam_url rtsp://user:pass@ip:port/Streaming/channels/102
movie_output on
movie_passthrough on
movie_extpipe_use on
movie_extpipe ffmpeg -y -f rawvideo -pix_fmt yuv420p -video_size %wx%h -framerate %fps -i pipe:0 -vcodec libx264 -preset ultrafast -f mp4 %f.mp4

but if you don't use ext_pipe, you will get a video from netcam_high_url @ 2304x1296
netcam_high_url rtsp://user:pass@ip:port/Streaming/channels/101
netcam_url rtsp://user:pass@ip:port/Streaming/channels/102
movie_output on
movie_passthrough on
movie_extpipe_use off

do you know of a way to configure movie_extpipe to use the newcam_high_url? i've been unable to figure it out.

Edit: i should add that if you don't use netcam_high_url and only use netcam_url (with the higher resolution feed) everything works fine. with the caveat of processing all those extra motion pixels.

**Reply by tosiara** [reply, marked as answer]
No, extpipe is for the main stream that is used for motion detection.
If you need full res - you simply should not use extpipe and it will save you the full res using passthrough.
Alternatively, you can use high-res for motion detection and simply dosable passthrough. Then you will get high-res in extpipe.

Usually, movie_output, movie_extpipe and movie_passthrough are mutually exclusive, you only use one of them. But technically you can use them all at once

**Comment by feelingwalnut** [comment]
thank you
